proposals/0000-dollar-type-operator-as-default.rst

```diff
@@ -0,0 +1,45 @@
+.. proposal-number::
+
+.. trac-ticket::
+
+.. implemented::
+
+Dollar type operator as default
+===============================
+
+Define the dollar type operator (`type a $ b = a b`) as a default when using
+the `TypeOperators` extension.
+
+Motivation
+----------
+
+Though, I have admittedly only used the `TypeOperators` extension once, I
+think that the main reason it is used is to have the function application
+operator available in type signatures. That's what I used it for.
+
+Proposed Change
+---------------
+
+When using the `TypeOperators` language extension, we should be able
```

Comment:
Inline code elements are demarcated by double-ticks in ReST, not single-ticks like in Markdown